convert_ucd_list_to_tex.py

```diff
@@ -11,7 +11,8 @@ def ucd_to_tex(file):
     with open(file, 'r') as f:
         lines = f.readlines()
         for cur_line in lines:
-            convert_line(cur_line)
+            if not cur_line.startswith("#"):
+                 convert_line(cur_line)
```

Comment:
The text above says "Comment rows start" -- I *think* you mean "comment lines", in which case this code would match the text.  The current text, however, sounds more like "delete all chars starting with a hash regardless of where it starts within a line".  We'll have to choose one or the other.

Reply:
ok for "Comment lines".

Reply:
corrections are included in #PR 80